lib/resources/r4/appointment.yaml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
this file is used when building the terminology bindings table

to actually consume it, we'd have to add this to appointment.md:

```
## Terminology Bindings

<%= terminology_table(:appointment, :r4) %>
```

But, we shouldn't actually need any of this content. We definitely don't need a terminology binding for resourceType, and since status has a required binding on the HL7 spec, we don't need to inform consumers what system we support - they have to use values from the required system

Reply:
dropped this file for PATCH, https://github.com/cerner/fhir.cerner.com/pull/257/commits/1b4f8ec57aee0a033fdbaeabdc7125f26d9a97c7#diff-fbb215a73c336ca1ac34d91de2fb411eL1